### DISCUSSION TITLE
Testnet Functionality

### DISCUSSION THREAD

**Discussion by ghost** [tracer-protocol/perpetual-pools-contracts#285] [Ideas]
### Moving Discussion from https://github.com/tracer-protocol/perpetual-pools-contracts/discussions/253

Some tasks identified in Discussion to date, this will spawn some child issues mostl ikely, so far I have

- [ ] [contracts deployed](https://github.com/tracer-protocol/perpetual-pools-contracts/issues/223 )
- [ ] scripted bots interacting with the contracts
- [ ] output logs of performance
- [ ] interface users can interact with
- [ ] spot farmer bot

**Comment by sporejack** [comment]
Related: #223

**Comment by ghost** [comment]
@mynameuhh @issy-fleming @greenbookwebb @Paul4912 @raymogg @sporejack Can we use this thread to help scope out the above items?
